input/fsh/SD_Extensions.fsh

```diff
@@ -59,9 +63,38 @@ Description: "Dose parameters prescribed for one radiotherapy volume."
   * ^short = "Number of Prescribed Fractions"
   * ^definition = "The number of Fraction in which dose to this volume is prescribed. See also extension RadiotherapyFractionsPrescribed which is used instead if fractions are not per volume, e.g. in Phase Prescriptions or Plans."
 
+Extension: RadiotherapyDosePlannedToVolume
+Id: codexrt-radiotherapy-dose-planned-to-volume
+Title: "Dose Planned to Volume"
+Description: "Dose parameters planned for one radiotherapy volume."
+* . ^short = "Planned Dose to a Dose Reference"
+* extension contains
+    volume 1..1 MS and
+    totalDosePlanned 0..1 MS and
+    fractionsPlanned 0..1 MS and
+    fractionDosePlanned 0..1 MS
+* extension[volume]
+  * value[x] only Reference(RadiotherapyVolume)
+  * ^short = "Volume to which Dose is Planned"
+  * ^definition = "A BodyStructure resource representing the body structure treated, for example, Chest Wall Lymph Nodes."
+* extension[totalDosePlanned]
+  * value[x] only Quantity
+  * valueQuantity = UCUM#cGy
+  * ^short = "Total Planned Radiation Dose"
+  * ^definition = "The total dose planned to this volume within the scope of this ServiceRequest."
+* extension[fractionsPlanned]
+  * value[x] only positiveInt
+  * ^short = "Number of Planned Fractions"
+  * ^definition = "The number of Fraction in which dose to this volume is planned. See also extension RadiotherapyFractionsPlanned which is used instead if fractions are the same for all volumes, i.e. in Planned Phases or Plans."
```

Comment:
"The number of Fraction in which dose to this volume is planned"=>
  "The planned number of fractions to deliver the dose"?

Reply:
> "The number of Fraction in which dose to this volume is planned"=> "The planned number of fractions to deliver the dose"?
Changed to "The planned number of Fractions to deliver the dose", and similar for "prescribed number of Fractions" in the respective extensions.